### ISSUE TITLE
Silent fail when network is big

### ISSUE THREAD

**Issue by mxrguspxrt** [tensorflow/tensorflow#2242]
### Steps to reproduce
1. Build a 10 layer hidden network with 2400 neurons in each layer 
2. Have data with correlation
3. Accuracy will be 1 / n_classes_outputs for each training step. 

### What have you tried?
1. If you will have less neurons and layers, everything works

I presume that there is a issue with memory and setting weights, but a error message would be nice.

**Comment by zheng-xq** [comment]
Could you post your environment, and a small code segment to reproduce the problem? Also please upload the log file so we can check if there is something meaningful there.

**Comment by mxrguspxrt** [comment]
Using Ubuntu, pip installation and latest TensorFlow (0.8.0). The issue seems to be, that my biases and weights go all to NaN objects, but I have no idea why. I remember gotting similar error in the past, then Exception was thrown, now it is silent and training goes to end. Everything works ok, if the network is smaller.

**Comment by girving** [comment]
@mxrguspxrt: Exploding to nans is standard with high learning rate is standard behavior for neural networks.  If you're seeing a rapidly exploding network with a very small network, there might be an issue, but as in I'm going to close this as (unfortunately) intended behavior.  Please comment if you think it should be reopened.
